node/service/src/chain_spec.rs

```diff
@@ -163,16 +160,25 @@ pub fn cere_dev_genesis(
 
 	const ENDOWMENT: Balance = 10_000_000_000 * TEST_UNITS;
 	const STASH: Balance = ENDOWMENT / 1000;
+	const MAX_POOLS: u32 = 16;
+	const MAX_MEMBERS_PER_POOL: u32 = 32;
+	const MAX_MEMBERS: u32 = MAX_POOLS * MAX_MEMBERS_PER_POOL;
+	const MULTIPLIER: u128 = 1_000_000_000_000_000_000;
```

Comment:
Is it equal to what we currently have in the runtime file or in a live network genesis config?